cmds/field/field.go

```diff
@@ -0,0 +1,366 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"unicode"
+	"unicode/utf8"
+)
+
+type frange struct {
+	begin int
+	end   int
+}
+
+const nf = 0x7FFFFFFF
```

Comment:
`nf` has long provenance; it comes from the special AWK variable 'NF', which stands for 'Number of Fields' in the input. So $NF would be the last field. I changed the constant to `LastField`.